README.md

```diff
@@ -7,7 +7,7 @@ provides a simulated micro:bit REPL in the browser.
 To build, first fetch the submodules (don't use recursive fetch):
 
     $ git submodule update --init lib/micropython-microbit-v2
-    $ git submodule update --init lib/micropython-microbit-v2/lib/micropython
+    $ (cd lib/micropython-microbit-v2 && git submodule update --init lib/micropython)
```

Comment:
You can also do `git -C lib/micropython-microbit-v2 submodule update --init lib/micropython` if you want it in one command.

But as you have it here is also good!

Reply:
Thanks, updated to your suggestion.